java_gen/java_type.py

```diff
@@ -684,6 +684,8 @@ def gen_fixed_length_string_jtype(length):
 
         'of_oxm_tcp_flags' : { 'value' : u16obj },
         'of_oxm_tcp_flags_masked' : { 'value' : u16obj, 'value_mask' : u16obj },
+        'of_oxm_ovs_tcp_flags' : { 'value' : u16obj },
+        'of_oxm_ovs_tcp_flags_masked' : { 'value' : u16obj, 'value_mask' : u16obj }
```

Comment:
Sorry for that. should I update it and re-pull? Or can you just modify it directly?